Workbooks/SapMonitor/SapHanaInfrastructure/SapHanaInfrastructure.workbook

```diff
@@ -0,0 +1,376 @@
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let startDate = startofday(ago(7d)); // go back in time nn days\r\nlet endDate = now(); // what is the date now\r\nlet projectTo = now()+7d; // project forward nn days\r\nlet projectForward = 7; // must be same as projectTo value\r\nSapHana_LoadHistory_CL\r\n|where TimeGenerated > ago(7d)\r\n| make-series avgCPU=avg(CPU_d) default=0 on UTC_TIMESTAMP_t in range(ago(7d),projectTo, 1h) by HOST_s\r\n| extend SizeForecast = series_decompose_forecast(avgCPU, projectForward*24)",
+        "size": 0,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+          "{LogAnalyticsWorkspaceArmId}"
+        ],
+        "visualization": "linechart",
+        "graphSettings": {
+          "type": 0
+        }
+      },
+      "name": "query - 7"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "Memory % Used  vs Forecast 7 Days"
+      },
+      "name": "text - 12"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let startDate = startofday(ago(7d)); // go back in time nn days\r\nlet endDate = now(); // what is the date now\r\nlet projectTo = now()+7d; // project forward nn days\r\nlet projectForward = 7; // must be same as projectTo value\r\nSapHana_Infra_CL\r\n|where TimeGenerated > ago(7d)\r\n| make-series avgMEM=avg(MEMORY_USED_d/MEMORY_SIZE_d)*100 default=0 on TimeGenerated in range(ago(7d),projectTo, 1h) by HOST_s\r\n| extend SizeForecast = series_decompose_forecast(avgMEM, projectForward*24)",
+        "size": 0,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "linechart",
+        "graphSettings": {
+          "type": 0
+        }
+      },
+      "name": "query - 7 - Copy"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "Disk Used vs Forecast 7 Days"
+      },
+      "name": "text - 14"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let startDate = startofday(ago(7d)); // go back in time nn days\r\nlet endDate = now(); // what is the date now\r\nlet projectTo = now()+7d; // project forward nn days\r\nlet projectForward = 7; // must be same as projectTo value\r\nSapHana_LoadHistory_CL\r\n|where TimeGenerated > ago(7d)\r\n| make-series avgDiskUsed=avg(DISK_USED_d) default=0 on UTC_TIMESTAMP_t in range(ago(7d),projectTo, 1h) by HOST_s\r\n| extend SizeForecast = series_decompose_forecast(avgDiskUsed, projectForward*24)",
+        "size": 0,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+          "{LogAnalyticsWorkspaceArmId}"
+        ],
+        "visualization": "linechart",
+        "graphSettings": {
+          "type": 0
+        }
+      },
+      "name": "query - 7 - Copy - Copy"
```

Comment:
while not *necessary*, we also suggest setting step titles in the individual steps in the template as well. the step names are useful for people who might be editing them later, and the step names are used in telemetry to identify which queries in your template are passing/failing/taking time.  we're working on exposing query performance telemetry as well, see the wiki in teams for onboarding instructions there.